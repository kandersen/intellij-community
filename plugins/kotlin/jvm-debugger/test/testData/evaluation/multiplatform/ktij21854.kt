

// MODULE: common
// FILE: common.kt
// PLATFORM: common
fun test() {
    val x = 1
    fun bar(): Int {
        val y = x
        return y
    }
    val c = C()

    //Breakpoint1
    val a = 0
}

class C {
    val local: String = ""
}

// ADDITIONAL_BREAKPOINT: common.kt / Breakpoint1 / line / 1

// EXPRESSION: c.local
// RESULT: "": Ljava/lang/String;

// MODULE: jvm
// FILE: jvm.kt
// PLATFORM: jvm
fun main() {
    test()
}
