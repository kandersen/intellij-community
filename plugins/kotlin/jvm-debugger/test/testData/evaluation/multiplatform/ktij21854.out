LineBreakpoint created at jvm.kt:15
Run Java
Connected to the target VM
jvm.kt:15
Compile bytecode for c.local
Disconnected from the target VM

Process finished with exit code 0
